### ISSUE TITLE
The application crashes when I clicked on Toolbox

### ISSUE THREAD

**Issue by juan-pablo-giron** [josegomez/epicor-editor-helper#23]
Hi @josegomez,

I want to thank you for your effort to build this application, it is great. Tried testing the application using Visual Studio 19 (free license). I could download a customization but when I click the ToolBox button I get the following error:

! [image] (https://user-images.githubusercontent.com/19398623/106638140-8c90fa80-6551-11eb-9838-e2a157f29327.png)

I don't know how I can solve this problem. I tried to repair my Visual Studio and reinstall your application, but the same issue appears again.

Thanks again !!! and hope that my issue will have a solution

**Comment by josegomez** [comment]
Try renaming your customization without periods in it, 
What version of Epicor is this?

**Comment by juan-pablo-giron** [comment]
I @josegomez ,

thanks for your response. I am using Epicor 10.2.500 version.

**Comment by josegomez** [comment]
ok same version I have, try re-naming your customization. Maybe those V.1.1.XX is causing issues.

**Comment by juan-pablo-giron** [comment]
I'm sorry, appears the same problem.

![image](https://user-images.githubusercontent.com/19398623/106639950-7e43de00-6553-11eb-9068-5168cf8e2843.png)

**Comment by josegomez** [comment]
You are clicking the toolbox with the project open right? I believe you have to have the customization downloaded / open to use the toolbox.

**Comment by juan-pablo-giron** [comment]
yes, I downloaded the project and immediately I try to open the ToolBox. I think that I am missing an library or package but I dont what.

**Comment by juan-pablo-giron** [comment]
@josegomez, This is the installation log, maybe it can give us a light.

![image](https://user-images.githubusercontent.com/19398623/106647400-dfbc7a80-655c-11eb-8a4e-2d27aadeccec.png)

**Comment by juan-pablo-giron** [comment]
Hello @josegomez,

Now the issue is the following: when I clicked on Toolbox Visual Studio restarts and doesn't show the Toolbox windows.
